tools/autograph-client/client.go

```diff
@@ -146,6 +150,7 @@ examples:
 	flag.IntVar(&maxworkers, "m", 1, "maximum number of parallel workers")
 	flag.StringVar(&cn, "cn", "", "when signing XPI, sets the CN to the add-on ID")
 	flag.IntVar(&sa, "sa", 0, "when signing MAR hashes, sets the Signature Algorithm")
+	flag.StringVar(&rsapssHash, "wa", "base64(sha1(data))", "for RSA-PSS Base64 SHA1 hash to sign using the /sign/hash endpoint")
```

Comment:
why is this a separate flag instead of just using the `-a` parameter?

Reply:
`-a` says it's for the sha256 hash